### ISSUE TITLE
Inference with pre-trained wav2vec2-base-960H - transcription is mostly non-existing words

### ISSUE THREAD

**Issue by mihaelagrigore** [facebookresearch/fairseq#5201]
I am trying to do inference using the pretrained wav2vec2-base-960H on a couple of audio files on English speakers conversation.

The decoded text looks like mostly non-words.
I was wondering how come, maybe I am doing something wrong.

The code I’m using is the most basic inference from the [wav2vec github page example](https://github.com/facebookresearch/fairseq/blob/main/examples/wav2vec/README.md)

`
import soundfile as sf
import torch
from datasets import load_dataset
from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor

processor = Wav2Vec2Processor.from_pretrained("facebook/wav2vec2-base-960h")
model = Wav2Vec2ForCTC.from_pretrained("facebook/wav2vec2-base-960h")

librispeech_samples_ds = load_dataset("patrickvonplaten/librispeech_asr_dummy", "clean", split="validation")

audio_input, sample_rate = sf.read(librispeech_samples_ds[0]["file"])

input_values = processor(audio_input, sampling_rate=sample_rate, return_tensors="pt").input_values  

logits = model(input_values).logits # this is the line where OOM appears
predicted_ids = torch.argmax(logits, dim=-1)

transcription = processor.decode(predicted_ids[0])
`

The output I get does not look like English words for a large part of it.

![image](https://github.com/facebookresearch/fairseq/assets/38474985/ec54b3b8-e27d-4a55-8c38-cd10a0f9e57d)

I suspect this is not a normal behavior for a tool that got the WER reported in their research papers.

And many other tools are being benchmarked against wav2vec2, so I’m very surprised this is actually a state of the art transcription that get one of the lowest WERs.

As can be been in the image of the transcription variable contents, most words are not real words.

Is there something that I am missing ? Does this transcription still need some processing ?
